linera-service/src/linera/main.rs

```diff
@@ -346,6 +346,7 @@ impl Runnable for Job {
                                         ValidatorState {
                                             network_address: address,
                                             votes,
+                                            version_info: Default::default(),
```

Comment:
Also unsure about this.  Surely we should be getting this by querying the validator in question.